themes/le-2025/layouts/partials/components/accordion-item.html

```diff
@@ -1,6 +1,6 @@
 <div class="accordion-item rounded-lg bg-gray-200 overflow-hidden {{ if not .last }}mb-4{{ end }}{{ if .is_open }} is-open{{ end }}">
   <button type="button" class="accordion-button w-full px-6 py-4 flex items-center gap-3  hover:bg-gray-300{{ if .is_open }} active{{ end }}" id="{{ .id }}-accordion">
-    <img src="/images/le-logo-lockonly.svg" class="w-5 h-5 transform -translate-y-0.5">
+    <img src="/images/le-logo-lockonly.svg" class="w-5 h-5 transform -translate-y-0.5" alt=""/>
```

Comment:
Shouldn't there be alt text here? I understand that this might get us a better score, but isn't it meaningless without actual text? Or is this blank because semantically the image is meaningless?

Reply:
This image is decorative, so there is no need to add a description.